examples/web_agent/How to schedule worker.md

```diff
@@ -0,0 +1,32 @@
+# How to run automatically the worker script
+
+In case the site you're acessing requires login, we suggest you hardcode the profile_id in the worker so it runs automatically.
+Just edit this line in worker.py
+
+`profile_id = input("Enter a profileId (or press Enter to skip): ").strip()`
+
+# Crontab (Linux, Mac)
+
+To schedule the worker with crontab you can create a cronjob with
+
+`crontab -e`
+
+Example of how to make the worker run every day
+
+`@daily python [PATH TO web_agent]/worker.py`
+
+Full documentation
+
+https://cloud.google.com/scheduler/docs/configuring/cron-job-schedules?hl=pt-br
```

Comment:
It's linking to the Portuguese version. Let's replace with the english version: https://cloud.google.com/scheduler/docs/configuring/cron-job-schedules

Reply:
Done